app/experimenter/legacy-ui/templates/experiments/nimbus-ui.html

```diff
@@ -0,0 +1,38 @@
+{% load static %}
+{% load experiment_extras %}
+
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="utf-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+  {% if nimbus_dev %}
+    <link rel="icon" href="http://localhost:3000/favicon.ico" />
```

Comment:
If this is the same global `favicon.ico` that the rest of the site uses, this should always be available in `static` shouldn't it?

Reply:
HEY. DIS A DRAFT.

Reply:
![EarlyHealthyEsok-small](https://user-images.githubusercontent.com/119884/95134617-2fb33e00-0731-11eb-8c4e-b79a7261fde6.gif)